pkg/azurefile/utils.go

```diff
@@ -129,3 +131,21 @@ func isSupportedFsType(fsType string) bool {
 	}
 	return false
 }
+
+func checkAzureRPCError(shareName, accountName string, err error) (bool, error) {
+	if err != nil {
+		for k, v := range sleepTimeWithAzureRPCErrors {
+			if strings.Contains(err.Error(), k) {
+				if k == clientThrottled {
+					klog.Warningf("EnsureStorageAccount(%s) failed with error(%v), sleep %ds to retry", accountName, err, v)
```

Comment:
in this fuction, actually you don't know whether it's `EnsureStorageAccount` or `CreateFileShare`, what about print out message outside of this function?

Reply:
done.